.github/workflows/tests.yml

```diff
@@ -37,17 +37,13 @@ jobs:
         uses: actions/cache@v3.3.1
         with:
           path: ${{ steps.composercache.outputs.dir }}
-          key: "php-${{ matrix.php-version }}-${{ matrix.dependencies }}-${{ hashFiles('**/composer.lock') }}"
-          restore-keys: "php-${{ matrix.php-version }}-${{ matrix.dependencies }}-${{ hashFiles('**/composer.lock') }}"
```

Comment:
`actions/cache` automatically use the value for `key` as a restore key. The `restore-keys` option allows to specify additional fallback keys.